unicodetools/org/unicode/text/tools/GenerateSubtagNames.java

```diff
@@ -16,11 +18,31 @@
 
 public class GenerateSubtagNames {
     public static void main(String[] args) {
+    	int n = generate(System.out);
```

Comment:
Let's please stick with an indent of 4 unless a file is already consistently different.